packages/angular/common/src/index.ts

```diff
@@ -25,7 +25,7 @@ export { IonPopover } from './overlays/popover';
 export { IonModal } from './overlays/modal';
 
 export { IonRouterOutlet, provideComponentInputBinding } from './directives/navigation/router-outlet';
-export type { StackEvent } from './directives/navigation/stack-utils';
+export type { StackWillChangeEvent } from './directives/navigation/stack-utils';
```

Comment:
This doesn't seem to be utilized outside of the common package, so we probably won't need this.